src/Http/Http.Features/src/IHttpRequestCanHaveBodyFeature.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    /// <summary>
+    /// Used to indicate if the request may have a body.
+    /// </summary>
+    public interface IHttpRequestCanHaveBodyFeature
```

Comment:
`IHttpBodyDetectionFeature` I guess we can discuss in API review.

Reply:
`IHttpRequestBodyDetectionFeature`?

I'll schedule something tomorrow, I want to get this in before Monday.